Libraries/Scripting/Sources/Scripts.swift

```diff
@@ -26,6 +26,8 @@ public final class Scripts {
         userScriptNames[$1] = $0
         return script
       }
+    //  sort file list case-insensitive for user display
+      .sorted(by: {$0.name.lowercased() < $1.name.lowercased()
```

Comment:
We usually do this: https://github.com/drawthingsai/draw-things-community/blob/main/Libraries/ModelZoo/Sources/ModelZoo.swift#L653 to avoid lowercased ambiguities (i.e. go beyond English alphabet, lowercased can have multiple meanings).

Reply:
good to know. I was sure there's a more universal compare.